lib/Conversion/TosaToEmitC/TosaToEmitC.cpp

```diff
@@ -610,6 +641,36 @@ class ArithmeticRightShiftOpConversion
   }
 };
 
+/// Convert `tosa.select` into an `emitc.call` operation.
+class SelectOpConversion : public OpConversionPattern<tosa::SelectOp> {
+  using OpConversionPattern<tosa::SelectOp>::OpConversionPattern;
+
+public:
+  SelectOpConversion(MLIRContext *ctx)
+      : OpConversionPattern<tosa::SelectOp>(ctx) {}
+
+private:
+  LogicalResult
+  matchAndRewrite(tosa::SelectOp selectOp, OpAdaptor adaptor,
+                  ConversionPatternRewriter &rewriter) const override {
+    StringRef funcName = "emitc::tosa::select";
+    StringAttr callee = rewriter.getStringAttr(funcName);
+
+    ArrayAttr args;
+    ArrayAttr templateArgs;
+
+    SmallVector<Value, 3> broadcastedOperands =
+        createBroadcastOpIfNeeded<3>(selectOp, adaptor, rewriter);
+
+    rewriter.replaceOpWithNewOp<emitc::CallOp>(
+        selectOp, selectOp.getType(), callee, args, templateArgs,
+        ValueRange({broadcastedOperands[0], broadcastedOperands[1],
```

Comment:
Aren't these all of broadcastedOperands? I think a ValueRange should automatically be constructed here.